src/view/undeploy.view.ts

```diff
@@ -121,8 +121,10 @@ export class UndeployView {
                     deployed = await waitUntilLambdaIsUndeployed();
                     /* istanbul ignore else */
                     if (deployed) {
-                      await clearIntervalAsync(timer);
-                      resolve();
+                      (async () => {
```

Comment:
can you add a comment a in the code explaining why this was added?

Reply:
added the comment